### PR TITLE
fixes: adds e2e test script to enhance local testing & workflow for self-hosted runner

### PR DESCRIPTION
**Pull request by JBAhire**
## Description
This PR, 
- adds a script to test Hypertrace helm deployment e2e locally
- Adds workflow for e2e test which is disabled in lack of self-hosted runner.
- Update gateway service port in helm values to fix broken kube setup
- Cleanup scripts directory

### Checklist:
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: kubernetes/hypertrace.sh [hunk 1]
```diff
@@ -133,24 +133,8 @@ case "$subcommand" in
...
-            No )
-              echo "[INFO] Uninstall cancelled."
-              exit;;
-        esac
-    done
-    ;;
-  *)
-    echo "[ERROR] Unknown command: ${subcommand}"
-    usage
+    clean
```
Comment (kotharironak) [inline kubernetes/hypertrace.sh:136]: why did you remove this from here?

Comment (JBAhire) [reply kubernetes/hypertrace.sh:136]: This was for workflow. As you can't provide user input there, this was to make that uninstall simpler.

File: kubernetes/hypertrace.sh [hunk 2]
```diff
@@ -154,3 +154,4 @@ case "$subcommand" in
     usage
     ;;
 esac
+
```
Comment (kotharironak) [inline kubernetes/hypertrace.sh:157]:
```suggestion
```

**Review by kotharironak — APPROVED** [review]
